pages/03-core/dual-axis-chart.page.tsx

```diff
@@ -0,0 +1,142 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: I think this page is not needed yet, as there is currently no proper support for dual-axis